app/core/views.py

```diff
@@ -9,11 +9,16 @@
 from django.contrib import messages
 from django.core.paginator import Paginator, EmptyPage, PageNotAnInteger
 from django.core.urlresolvers import reverse
+from django.core.mail import send_mail
+from django.core.exceptions import ObjectDoesNotExist
 from django.db.models import Q, Sum
 from django.http import HttpResponse, JsonResponse, HttpResponseRedirect
 from django.shortcuts import render, get_object_or_404, redirect
 from django.utils.encoding import force_unicode
 from django.utils.translation import ugettext_lazy as _
+from django.views.generic import FormView
+from django.views import generic
```

Comment:
Instead of importing `FormView` directly but importing `generic.TemplateView` separately, try `from django.views.generic import FormView, TemplateView`